projects/tm_example/hello_tm_apical_tiebreak.py

```diff
@@ -0,0 +1,153 @@
+# representing the sequence A -> B -> C -> D -> E
+x = numpy.zeros((5, tm.numberOfColumns()), dtype="uint32")
+x[0, 0:10] = 1    # Input SDR representing "A", corresponding to columns 0-9
+x[1, 10:20] = 1   # Input SDR representing "B", corresponding to columns 10-19
+x[2, 20:30] = 1   # Input SDR representing "C", corresponding to columns 20-29
+x[3, 30:40] = 1   # Input SDR representing "D", corresponding to columns 30-39
+x[4, 40:50] = 1   # Input SDR representing "E", corresponding to columns 40-49
+
+
+# Step 3: send this simple sequence to the temporal memory for learning
+# We repeat the sequence 10 times
+for i in range(10):
+
+  # Send each letter in the sequence in order
+  for j in range(5):
+    activeColumns = [i for i, j in zip(count(), x[j]) if j == 1]
+
+    # The compute method performs one step of learning and/or inference. Note:
+    # here we just perform learning but you can perform prediction/inference and
+    # learning in the same step if you want (online learning).
+
+    tm.compute(activeColumns, apicalInput=(), apicalGrowthCandidates=None, learn=True)
+
+    # The following print statements can be ignored.
+    # Useful for tracing internal states
+
+    print("active cells " + str(tm.getActiveCells()))
+    print("predictive cells " + str(tm.getNextPredictedCells()))
+    print("winner cells " + str(tm.getWinnerCells()))
+    print("# of active segments " + str(numpy.shape(tm.getActiveBasalSegments())[0]))
+
+  # The reset command tells the TM that a sequence just ended and essentially
+  # zeros out all the states. It is not strictly necessary but it's a bit
+  # messier without resets, and the TM learns quicker with resets.
+  tm.reset()
+
+
+#######################################################################
+#
+# Step 3: send the same sequence of vectors and look at predictions made by
+# temporal memory
+for j in range(5):
+  print "\n\n--------","ABCDE"[j],"-----------"
+  print "Raw input vector : " + formatRow(x[j])
+  activeColumns = [i for i, j in zip(count(), x[j]) if j == 1]
+  # Send each vector to the TM, with learning turned off
+  tm.compute(activeColumns, apicalInput=(), apicalGrowthCandidates=None, learn=False)
+
+  # The following print statements prints out the active cells, predictive
+  # cells, active segments and winner cells.
+  #
+  # What you should notice is that the columns where active state is 1
+  # represent the SDR for the current input pattern and the columns where
+  # predicted state is 1 represent the SDR for the next expected pattern
+  print "\nAll the active and predicted cells:"
+
+
+  print("active cells " + str(tm.getActiveCells()))
+  print("predictive cells " + str(tm.getNextPredictedCells()))
+  print("winner cells " + str(tm.getWinnerCells()))
+  print("# of active segments " + str(numpy.shape(tm.getActiveBasalSegments())[0]))
+
+  cells_per_column = tm.getCellsPerColumn()
+  activeColumnsIndeces = [int(i / cells_per_column) for i in tm.getActiveCells()]
```

Comment:
Incorrect spelling of "Indices" (also in rest of file). Yes, I know this is carried over from the original example 😄